src/drive/storage/index.ts

```diff
@@ -91,13 +92,15 @@ export class Storage {
    * Fetches & returns the contents of a specific folder
    * @param folderId
    */
-  public getFolderContent(folderId: number): [
+  public getFolderContent(folderId: number, trash: (boolean | null) = null): [
```

Comment:
Using ? for an optional parameter could be a better option considering that the parameter is in fact, optional